CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.3
+
+- [FP-2802 - Revert lib-ide to the version before the MUI / tty changes](https://movai.atlassian.net/browse/FP-2802)
```

Comment:
@diasdm and @TiagoLopesQA  we need a decision here.

Reply:
According to the discussion with @diasdm you should start using the new nomenclature:
`[ticket_number](ticket_link): ticket small description`

Reply:
```suggestion
- [FP-2802](https://movai.atlassian.net/browse/FP-2802): Revert lib-ide to the version before the MUI / tty changes
```